app/assets/stylesheets/fuel/_post-list.scss

```diff
@@ -0,0 +1,69 @@
+// Page Siebar Component
+// ========================================
+
+
+// Variables
+$resource-item-border:       1px solid $grey-light;
+$resource-item-padding:      15px 20px;
+
+$resource-hover-background:  $white-dark;
+
+$resource-active-background: $white-dark;
+$resource-active-color:      $blue-base;
+$resource-date-color:        $grey-dark;
+
+// Structure
+.resource-list {
+  @include rem(padding-bottom, 50px);
+  height: 100%;
+  overflow-x: hidden;
```

Comment:
Properties should be ordered height, -webkit-overflow-scrolling, overflow-x, overflow-y